packages/ui/src/views/repo/pull-request/details/components/conversation/pull-request-comment-box.tsx

```diff
@@ -61,11 +61,7 @@ export const PullRequestCommentBox = ({
   }
 
   const avatar = useMemo(() => {
```

Comment:
This memo looks kinda useless. Might make sense to just inline the avatar where it's needed.